### PR TITLE
MGMT-2592: filtering media disks from the inventory

### PR DESCRIPTION
**Pull request by osherdp**
* All devices that contain ISO content are removed from the host's inventory
* Let ``ghw`` library use the ``/host`` root directory, to capture host's HW instead of the "guest HW"

### CODE REVIEW

**Comment by filanov** [comment]
cc: @ronniel1 @yevgeny-shnaidman

**Comment by osherdp** [comment]
depends on https://github.com/openshift/assisted-service/pull/627

File: src/inventory/disks.go [hunk 1]
```diff
@@ -76,18 +76,30 @@ func unknownToEmpty(value string) string {
...
 	ret := make([]*models.Disk, 0)
-	blockInfo, err := d.dependencies.Block()
+	blockInfo, err := d.dependencies.Block(ghw.WithChroot("/host"))
 	if err != nil {
 		logrus.WithError(err).Warnf("While getting disks info")
 		return ret
 	}
 	for _, disk := range blockInfo.Disks {
-		if disk.IsRemovable || disk.SizeBytes == 0 ||
+		if disk.IsRemovable || disk.SizeBytes == 0 || isInstallationMedia(disk) ||
```
Comment (empovit) [inline src/inventory/disks.go:98]: Are there unit-tests that test this branch?

Comment (osherdp) [reply src/inventory/disks.go:98]: Now there are :)

**Pull request merged**